### ISSUE TITLE
Build instructions and binary

### ISSUE THREAD

**Issue by danboid** [corthax/mdtracker#1]
Please add build instructions to the README. A screenshot and a pre-built binary would be great too.

Thanks

**Comment by corthax** [comment]
This is regular https://github.com/Stephane-D/SGDK/wiki/Setup-SGDK-with-CodeBlocks project.
Some binaries are in /out folder (*.bin). There are also releases 1.0, 1.1 releases.
Latest 1.2 is in /out/releases. Need to finish med x7 version, also the rom/save editing tool, but tracker itself already works fine.
I'll add screenshot.

**Comment by danboid** [comment]
Ah yes! 'out' :) I was only thinking bin as a UNIX guy :)

The README currently explains that situation thus:

"/out - latest wip rom here (rom.bin)"

rom is a bit vague to me. I'd maybe reword this to something like  "See /out/rom.bin for the latest build of mdtracker" maybe so that others don't miss it like I did.

**Comment by danboid** [comment]
There was of course a.out before we had elf binaries but still I missed it!
